models/marts/advertising/fct_ad_performance.sql

```diff
@@ -51,15 +51,15 @@ marketing_channels as (
             when lower(campaign) ilike '%display%' then 'display'
             when lower(campaign) ilike '%shopping%'
                 or lower(campaign) ilike '%cpc%' then 'paid search'
-            when utm_medium is null and utm_medium not in ('fb_retargeted', 'ig_retargeted') 
+            when utm_medium is null and utm_campaign not in ('fb_retargeted', 'ig_retargeted') 
                 then 'direct'
```

Comment:
@fkalkhalidi should any ads show up as `direct`? on the snowplow side this makes sense but on the ad side we are seeing facebook ads like `utm_campaign = KCP - Store - RT - KCPVC180D (HT_01) (VCEvent) (6-28-18)' show up as direct

```
select distinct
    platform, utm_medium, utm_campaign, campaign, channel
from dbt_adiaz.fct_ad_performance
limit 100
```

Reply:
Yes, we shouldn't have any ads show up as direct. 

Also, It seems to me that a number of our social and email campaigns have not been tagged with a utm_medium.

Reply:
@fkalkhalidi there are a number of facebook ads that don't have utm_source or utm_medium, but the majority of utm_campaigns show things like `fb`, `KCP - Store - Cold - LAL - EquipBuyersListLAL 1% (HT_01) (PurchaseEvent) (6-16-18)`,  `prospecting`, `LaborDay`, etc. how should we account for those?